elasticdl/python/collective_ops/communicator.py

```diff
@@ -0,0 +1,18 @@
+# TODO: This is dummy for now until the real implementation
+# has been open sourced
+
+from elasticdl.python.common.constants import CollectiveCommunicatorStatus
+
+
+class CollectiveCommunicator(object):
+    def __init__(self):
+        pass
+
+    def allreduce(self, data, op="MEAN"):
+        return CollectiveCommunicatorStatus.SUCCEEDED, data
+
+    def broadcast(self, from_worker_ip):
```

Comment:
How can the broadcast sender send some data using this method? `data` argument may also be needed.